spec/jobs/add_diaper_partner_job_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe AddDiaperPartnerJob, job: true do
+  describe "#perform" do
+    let(:partner) do
+      partner = create(:partner)
+      partner.organization.update!(invitation_text: 'Invitation')
+      partner
+    end
+
+    context "successful diaper client request" do
+      before do
+        response = double("Response", value: Net::HTTPSuccess)
+        allow(DiaperPartnerClient).to receive(:add).and_return(response)
+      end
+
+      it "invokes add method on diaper partner client with proper arguments" do
+        Sidekiq::Testing.inline! do
+          AddDiaperPartnerJob.perform_async(partner.id, email: 'test@test.com')
+
```

Comment:
We recently (last week) made some changes to the Jobs and that means we can't use Sidekiq API methods in the tests anymore. This test is failing, but the fix isn't too complicated:
Check out:
https://github.com/rubyforgood/diaper/blob/master/spec/jobs/update_diaper_partner_job_spec.rb#L50

For reference -- I encountered this error a bunch

Reply:
Thanks for the review, I updated the PR